dev/archery/archery/crossbow/cli.py

```diff
@@ -233,6 +233,27 @@ def latest_prefix(obj, prefix, fetch):
     click.echo(latest.branch)
 
 
+@crossbow.command()
+@click.argument('job-name', required=True)
+@click.pass_obj
+def save_report_data(obj, job_name):
+    """
+    Just print there the state of the job
+    """
+    output = obj['output']
+    
+    queue = obj['queue']
+    print(dir(queue))
+    
+    if fetch:
+        queue.fetch()
+
+    job = queue.get(job_name)
+    report = JsonReport(job=job)
+
+    report.show(output)
```

Comment:
I think we are still missing the implementation of `JsonReport.show`. But the real question that I don't yet have clear is _where_ should we ever see this output if `save-report-data` command is not yet attached to the nightly builds execution? Is there a way to try nightly build changes _before_ making them real?